tutorials/hello-kubernetes/README.md

```diff
@@ -293,7 +297,35 @@ Got a new order! Order ID: 3
 Successfully persisted state
 ```
 
-## Step 7 - Confirm successful persistence
+## Step 7 - Observe API call logs
+
+Now that the Node.js and Python applications are deployed, watch API call logs come through:
+
+Get the API call logs of the Node.js app:
+
+<!-- STEP
+expected_stdout_lines:
+  - 'level=info msg="HTTP API Called: POST /v1.0/state/statestore"'
+expected_stderr_lines:
+output_match_mode: substring
+name: Read nodeapp logs
+-->
+
+```bash
+POD=$(kubectl get pod -l app=node -o jsonpath="{.items[0].metadata.name}")
+kubectl logs $POD daprd
+```
+
+<!-- END_STEP -->
+
+When save state API calls are made, you should see logs similar to this:
+
+```
+time="2022-04-25T22:46:09.82121774Z" level=info msg="HTTP API Called: POST /v1.0/state/statestore" app_id=nodeapp instance=nodeapp-7dd6648dd4-7hpmh scope=dapr.runtime.http-info type=log ver=1.7.2
+time="2022-04-25T22:46:10.828764787Z" level=info msg="HTTP API Called: POST /v1.0/state/statestore" app_id=nodeapp instance=nodeapp-7dd6648dd4-7hpmh scope=dapr.runtime.http-info type=log ver=1.7.2
+```
```

Comment:
Also add this

"Get the API call logs of the python app:

kubectl logs --selector=app=python -c daprd --tail=-1